<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BAKoffeeClass</class>
 <widget class="QWidget" name="BAKoffeeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BAKoffee</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnColombia">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Colombia</string>
       </property>
       <property name="icon">
        <iconset resource="bakoffee.qrc">
         <normaloff>:/BAKoffee/Resources/coffee.png</normaloff>:/BAKoffee/Resources/coffee.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSumatra">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Sumatra</string>
       </property>
       <property name="icon">
        <iconset resource="bakoffee.qrc">
         <normaloff>:/BAKoffee/Resources/coffee.png</normaloff>:/BAKoffee/Resources/coffee.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeca">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>DÃ©ca</string>
       </property>
       <property name="icon">
        <iconset resource="bakoffee.qrc">
         <normaloff>:/BAKoffee/Resources/coffee.png</normaloff>:/BAKoffee/Resources/coffee.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEspresso">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Espresso</string>
       </property>
       <property name="icon">
        <iconset resource="bakoffee.qrc">
         <normaloff>:/BAKoffee/Resources/coffee.png</normaloff>:/BAKoffee/Resources/coffee.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btnPlusSucre">
         <property name="text">
          <string>Plus</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="bakoffee.qrc">:/BAKoffee/Resources/sugar.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinSucre">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnMoinsSucre">
         <property name="text">
          <string>Moins</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="btnPlusLait">
         <property name="text">
          <string>Plus</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="bakoffee.qrc">:/BAKoffee/Resources/milk.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinLait">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnMoinsLait">
         <property name="text">
          <string>Moins</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="btnPlusCaramel">
         <property name="text">
          <string>Plus</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="bakoffee.qrc">:/BAKoffee/Resources/caramel.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinCaramel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnMoinsCaramel">
         <property name="text">
          <string>Moins</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="btnPlusChantilly">
         <property name="text">
          <string>Plus</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>62</width>
             <height>64</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="bakoffee.qrc">:/BAKoffee/Resources/chantilly.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinChantilly">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnMoinsChantilly">
         <property name="text">
          <string>Moins</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnGo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(85, 170, 0, 139), stop:1 rgba(255, 255, 255, 142));
alternate-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(85, 170, 0, 139), stop:1 rgba(255, 255, 255, 142));
border-color: rgb(175, 175, 175);</string>
       </property>
       <property name="text">
        <string>Go !</string>
       </property>
       <property name="icon">
        <iconset resource="bakoffee.qrc">
         <normaloff>:/BAKoffee/Resources/coffee.png</normaloff>:/BAKoffee/Resources/coffee.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btnColombia</tabstop>
  <tabstop>btnSumatra</tabstop>
  <tabstop>btnDeca</tabstop>
  <tabstop>btnEspresso</tabstop>
  <tabstop>btnPlusSucre</tabstop>
  <tabstop>btnMoinsSucre</tabstop>
  <tabstop>btnPlusLait</tabstop>
  <tabstop>btnMoinsLait</tabstop>
  <tabstop>btnPlusCaramel</tabstop>
  <tabstop>btnMoinsCaramel</tabstop>
  <tabstop>btnPlusChantilly</tabstop>
  <tabstop>btnMoinsChantilly</tabstop>
 </tabstops>
 <resources>
  <include location="bakoffee.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnPlusSucre</sender>
   <signal>clicked()</signal>
   <receiver>spinSucre</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMoinsSucre</sender>
   <signal>clicked()</signal>
   <receiver>spinSucre</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPlusLait</sender>
   <signal>clicked()</signal>
   <receiver>spinLait</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMoinsLait</sender>
   <signal>clicked()</signal>
   <receiver>spinLait</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPlusCaramel</sender>
   <signal>clicked()</signal>
   <receiver>spinCaramel</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMoinsCaramel</sender>
   <signal>clicked()</signal>
   <receiver>spinCaramel</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPlusChantilly</sender>
   <signal>clicked()</signal>
   <receiver>spinChantilly</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMoinsChantilly</sender>
   <signal>clicked()</signal>
   <receiver>spinChantilly</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnGo</sender>
   <signal>clicked()</signal>
   <receiver>BAKoffeeClass</receiver>
   <slot>go()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnColombia</sender>
   <signal>clicked()</signal>
   <receiver>BAKoffeeClass</receiver>
   <slot>enableGo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSumatra</sender>
   <signal>clicked()</signal>
   <receiver>BAKoffeeClass</receiver>
   <slot>enableGo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDeca</sender>
   <signal>clicked()</signal>
   <receiver>BAKoffeeClass</receiver>
   <slot>enableGo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEspresso</sender>
   <signal>clicked()</signal>
   <receiver>BAKoffeeClass</receiver>
   <slot>enableGo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>go()</slot>
  <slot>enableGo()</slot>
 </slots>
</ui>
